bundles/org.openhab.binding.sonos/src/main/java/org/openhab/binding/sonos/internal/handler/ZonePlayerHandler.java

```diff
@@ -2351,20 +2386,20 @@ private void scheduleNotificationSound(final Command command) {
      * @param url in the format of //host/folder/filename.mp3
      */
     public void playNotificationSoundURI(Command notificationURL) {
-        if (notificationURL != null && notificationURL instanceof StringType) {
+        if (notificationURL instanceof StringType) {
             try {
                 ZonePlayerHandler coordinator = getCoordinatorHandler();
 
                 String currentURI = coordinator.getCurrentURI();
                 logger.debug("playNotificationSoundURI: currentURI {} metadata {}", currentURI,
                         coordinator.getCurrentURIMetadataAsString());
 
-                if (isPlayingStream(currentURI) || isPlayingRadioStartedByAmazonEcho(currentURI)
-                        || isPlayingRadio(currentURI)) {
+                if ((currentURI != null) && (isPlayingStream(currentURI)
+                        || isPlayingRadioStartedByAmazonEcho(currentURI) || isPlayingRadio(currentURI))) {
```

Comment:
```suggestion
                if (isPlayingStream(currentURI) || isPlayingRadioStartedByAmazonEcho(currentURI) 
                        || isPlayingRadio(currentURI)) {
```

Reply:
I probably added the test on currentURI to avoid a warning. To be checked again.

Reply:
Fixed.